content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+
+    The payee can always close a payment channel immediately after processing a claim. The payer can close the payment channel immediately if it has no XRP remaining in it.
+
+    Example of submitting a transaction requesting a channel to close:
+
+        {
+            "method": "submit",
+            "params": [{
+                "secret": "s████████████████████████████",
+                "tx_json": {
+                    "Account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                    "TransactionType": "PaymentChannelClaim",
+                    "Channel": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+                    "Flags": 2147614720
+                },
+                "fee_mult_max": 1000
+            }]
+        }
+
+    After the transaction is included in a validated ledger, either party can look up the currently-scheduled expiration of the channel using the `account_channels` method. Be sure to specify `"ledger_index": "validated"` to get data from the latest validated ledger version.
+
+    Example `account_channels` response:
+
+        {
+            "result": {
+                "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                "channels": [
+                    {
+                        "account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                        "amount": "100000000",
+                        "balance": "1000000",
+                        "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+                        "destination_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                        "destination_tag": 20170428,
+                        "expiration": 547073182,
+                        "public_key": "aB44YfzW24VDEJQ2UuLPV2PvqcPCSoLnL7y5M1EzhdW4LnK5xMS3",
+                        "public_key_hex": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+                        "settle_delay": 86400
+                    }
+                ],
+                "status": "success"
+            }
+        }
+
+    In this example, the `expiration` value 547073182 in [seconds since the Ripple Epoch](reference-rippled.html#specifying-time) maps to 2017-05-02T20:46:22Z, so any claims not redeemed by that time are no longer valid.
+
+9. **After the settlement delay has passed or the channel has reached its planned expiration time, the channel is expired.** Any further transaction that would affect the channel can only close it, returning unclaimed XRP to the payer.
+
+    The channel can remain on the ledger in an expired state indefinitely. This is because the ledger cannot change except as the results of a transaction, so _someone_ must send a transaction to cause the expired channel to close.
```

Comment:
@ryangyoung Payment channels are part of the payer's owner count, which effects the payer's reserve: https://ripple.com/build/reserves/